kong/clustering/compat/checkers.lua

```diff
@@ -67,6 +67,22 @@ local compatible_checkers = {
 
             has_update = true
           end
+
+          if config.model.provider == "mistral" and (
+            not config.model.options or
+            config.model.options == ngx.null or
+            not config.model.options.upstream_url or
+            config.model.options.upstream_url == ngx.null) then
```

Comment:
There should be a simpler way to express "config.model.options.upstream_url not exist)

Reply:
i hope there exists some syntax sugar like `config.model?.options?.upstream_url`. everytime i write this i feel like writing java.